wifiphisher/interfaces.py

```diff
@@ -0,0 +1,440 @@
+"""
+This module was made to handle all the interface related operations for
+Wifiphisher.py
+"""
+
+import subprocess
+import os
+import re
+
+DEVNULL = open(os.devnull, "w")
```

Comment:
You should import and use the DN from <a href="https://github.com/sophron/wifiphisher/blob/master/wifiphisher/constants.py#L17">constants.py</a> file.